source/templates/binding-element-attributes.md

```diff
@@ -79,23 +79,19 @@ Now the same handlebars code above renders the following HTML:
        type="text" data-toggle="tooltip" data-placement="bottom" title="Name">
 ```
 
-You can also automatically bind data attributes on the base view with the
-following:
+You can also automatically bind data and aria attributes on the base component with the following:
 
 ```javascript
-export default Ember.View.reopen({
-  init() {
-    this._super();
-    var self = this;
-
-    // bind attributes beginning with 'data-'
-    Ember.keys(this).forEach(function(key) {
-      if (key.substr(0, 5) === 'data-') {
-        self.get('attributeBindings').pushObject(key);
+export default Ember.Component.reopen({
```

Comment:
someone raised a question about exporting a `.reopen`, I think this snippet needs to be reviewed.

cc @mixonic @michaelrkn